core-bundle/src/Resources/contao/library/Contao/Folder.php

```diff
@@ -585,16 +585,18 @@ public static function scan($strFolder, $blnUncached=false): array
 		$arrReturn = array();
 
 		// Scan directory
-		foreach (scandir($strFolder, SCANDIR_SORT_ASCENDING) as $strFile)
+		if (is_dir($strFolder))
 		{
-			if ($strFile == '.' || $strFile == '..')
+			foreach (scandir($strFolder, SCANDIR_SORT_ASCENDING) as $strFile)
 			{
-				continue;
-			}
+				if ($strFile == '.' || $strFile == '..')
+				{
+					continue;
+				}
 
-			$arrReturn[] = $strFile;
+				$arrReturn[] = $strFile;
+			}
 		}
```

Comment:
```suggestion
		}

```

I wasn‘t allowed to fix this myself, so now you have to merge this. 😄